pkg/api/api_to_model.go

```diff
@@ -0,0 +1,165 @@
+package api
```

Comment:
Code that converts the GRPC API and sqlite DBs into the model isn't strictly required for `opm serve`. However it's used in `pkg/server/server_test.go` to test that both of the following scenarios result in the equivalent GRPC responses:
1. sqlite -> GRPC
2. sqlite -> model -> GRPC